site-src/guides/migrating-from-ingress.md

```diff
@@ -251,7 +251,7 @@ In contrast, Gateway API specifies how to merge rules and resolve conflicts:
 * A Gateway implementation must merge the routing rules from all HTTPRoutes
   attached to a listener.
 * Conflicts must be handled as
-  prescribed [here](/concepts/guidelines/#conflicts). For example, more specific
+  prescribed in [API Design Guide: Conflicts](/guides/api-design/#conflicts). For example, more specific
```

Comment:
The link appeared to redirect, so I've updated it, and then tried to pick a title based on the resolved page...